### DISCUSSION TITLE
Which 🇪🇺 capital city claims the title of being highest above sea level?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#760] [Galaxy Trivia] [answered]
**Bonus:** What is the height?

**Comment by peterbaccaro** [comment, marked as answer]
The capital city of Europe that **claims* the title of being highest above sea level is **Madrid**, the capital of Spain. Madrid is located in the centre of the Iberian Peninsula, at an elevation of approximately **667 meters (2,188 feet)** above sea level.
Madrid's high altitude is due to its location on the Spanish Central Plateau, which is surrounded by mountain ranges that rise to over 2,000 meters (6,500 feet) above sea level. 

**REFERENCE**
https://economictimes.indiatimes.com/magazines/travel/madrid-city-highest-capital-in-europe/articleshow/6230863.cms
![madrid](https://github.com/ParanoidUser/yolo/assets/74666079/8d353650-fed1-428d-a2c4-fa86fdb7b7ac)

On the continent of Europe, Andorra la Vella is higher than Madrid at 1,023 meters however **Andorra is not in the European Union** and the **🇪🇺 emoji represents the European Union**.

**Comment by SANKARAMDAS** [comment]
Andorra la Vella
Europe's highest capital is Andorra la Vella, at 1,023m.
Andorra la Vella, which is 1023 metres above sea level and the capital of Andorra.
